idb/postgres/postgres.go

```diff
@@ -1136,6 +1137,9 @@ ON CONFLICT (addr, assetid) DO UPDATE SET amount = account_asset.amount + EXCLUD
 					return fmt.Errorf("app delta apply err r=%d i=%d app=%d, %v", adelta.Round, adelta.Intra, adelta.AppIndex, err)
 				}
 			}
+			reverseDelta.ExtraProgramPages = adelta.ExtraProgramPages
```

Comment:
I think it should be `reverseDelta.ExtraProgramPages = state.ExtraProgramPages`.

Reply:
ops, fixed